strawberry/field.py

```diff
@@ -353,6 +356,7 @@ def is_async(self) -> bool:
 @overload
 def field(
     *,
+    field_class: Type[StrawberryField] = StrawberryField,
```

Comment:
Do we still need the `field_class` change when using `FieldExtension`?

Reply:
Oh, we don't need it anymore. Forgot to remove it from the PR :)